tokio-console/src/main.rs

```diff
@@ -127,6 +127,27 @@ async fn main() -> color_eyre::Result<()> {
                     .0
                     .push(Span::styled(" PAUSED", view.styles.fg(Color::Red)));
             }
+            let dropped_async_ops_state = state.async_ops_state().dropped_events();
+            if dropped_async_ops_state > 0 {
+                header_text.0.push(Span::styled(
+                    format!(" dropped async_ops_state: {}", dropped_async_ops_state),
+                    view.styles.fg(Color::Red),
+                ));
+            }
+            let dropped_tasks_state = state.tasks_state().dropped_events();
+            if dropped_tasks_state > 0 {
+                header_text.0.push(Span::styled(
+                    format!(" dropped tasks_state: {}", dropped_tasks_state),
+                    view.styles.fg(Color::Red),
+                ));
+            }
+            let dropped_resources_state = state.resources_state().dropped_events();
+            if dropped_resources_state > 0 {
+                header_text.0.push(Span::styled(
+                    format!(" dropped resources_state: {}", dropped_resources_state),
+                    view.styles.fg(Color::Red),
+                ));
+            }
```

Comment:
take it or leave it: we could probably save a little horizontal real estate by formatting the text like
```
dropped XXXX tasks, YYYY resources, ZZZZ async ops
```
or something?

instead of repeating the word "dropped" three times, we could check if any counter is non-zero, print a single "dropped", and then print the actual counts. not a blocker, though, as the current code should fit on most terminals.

Reply:
Good idea, updated.